package.json

```diff
@@ -42,10 +42,12 @@
     "@babel/preset-env": "7.20.2",
     "@babel/preset-typescript": "7.18.6",
     "@microsoft/api-extractor": "7.33.5",
+    "@rollup/plugin-commonjs": "^23.0.5",
```

Comment:
lmk if u want this + adding it to rollup config in a separate PR

Reply:
added as peerDep, removed this d102791cbb45db10d6d382918310a7ad9db17e22. Thanks for the tips, i'm a rollup noob!